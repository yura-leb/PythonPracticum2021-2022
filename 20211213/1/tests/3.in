[40, 104, 24, 248, 168, 120, 184, 8, 56, 136, 88, 72, 200, 216, 232, 152]
