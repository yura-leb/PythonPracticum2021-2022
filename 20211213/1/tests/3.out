[8, 24, 40, 56, 72, 88, 104, 120, 136, 152, 168, 184, 200, 216, 232, 248]
